Cellpose_Plugin_CellProfiler_Instructions/README.md

```diff
@@ -0,0 +1,97 @@
+# Cellpose Plugin for CellProfiler installation
+
+## For Linux:
+
+### Step 1 
+
+Create a `.yml` file using the template below from the [CellProfiler conda environment wiki page](https://github.com/CellProfiler/CellProfiler/wiki/Conda-Installation):
```

Comment:
Why not provide the `yml` rather than a manual step?

Reply:
No reason at all :laughing: For some reason I thought showing the file was a good idea. Easy change!